src/content/docs/concepts/security/permissions.mdx

```diff
@@ -0,0 +1,109 @@
+---
+title: Permissions
+---
+
+Permissions are descriptions of explicit privileges of commands.
+
+It can enable commands to be accessible in the frontend of a Tauri application.
+It can map scopes to commands and defines which commands are enabled.
+Permissions can enable or deny certain commands, define scopes or combine both.
+
+Permissions can be grouped as a set under a new identifier.
+This is called a permission set.
```

Comment:
There is both permission sets and capabilities that allows grouping of permissions. I think this is confusing, should we recommend a use case for each?